src/views/retro_active/RetroActiveView.module.scss

```diff
@@ -23,6 +23,9 @@
 
     @include assets.scrollbar;
 }
+.container::-webkit-scrollbar-thumb {
+    border-radius: 0px
+}
```

Comment:
```suggestion

    &::-webkit-scrollbar-thumb {
        border-radius: 0px
    }
}
```

Reply:
chyba można w ten sposób to ograć też

a no i jak podajesz 0 to nie musisz dopisywać px

Reply:
**GOOD LESSON** c: